### ISSUE TITLE
CloudFormation does not expose ExportName

### ISSUE THREAD

**Issue by rbu** [localstack/localstack#1408]
Given a CF template that defines outputs as named exports, I expect the "describe-stack" endpoint to include the exported names. With localstack, that is currently not the case.

Example template:
```yaml
  Outputs:
    VPCId:
      Value: {Ref: MyVPC}
      Export:
        Name: My-VPCId

  Resources:
    MyVPC:
      Properties:
        CidrBlock: 10.0.0.0/16
      Type: AWS::EC2::VPC

```

AWS:
```
$ aws cloudformation describe-stacks
...
                {
                    "OutputKey": "VPCId",
                    "OutputValue": "vpc-0123456789",
                    "ExportName": "My-VPCId"
                },

```

localstack:
```
$ aws cloudformation --endpoint-url=http://localhost:4581/ describe-stacks
...
                {
                    "OutputKey": "VPCId",
                    "OutputValue": "vpc-0123456789"
                },
...
```

**Comment by MadMouse** [comment]
Did you manage to resolve this issue?

 My serverless scripts depend on ExportName and fails to deploy.

**Comment by rbu** [comment]
> Did you manage to resolve this issue?
> 
> My serverless scripts depend on ExportName and fails to deploy.

I haven't worked on this. It's (one of) the blocker(s) to using localstack in one of my projects.

**Comment by whummer** [comment]
Thanks for reporting @rbu @MadMouse . This should be fixed in #2033 - can you please give it another try with the `latest` version? Please report here if the problem persists. Thanks
